### PR TITLE
Rename variables of Postmoogle to handle it as a bridge

### PR DESCRIPTION
**Pull request by luixxiul**
Follow-up to https://github.com/spantaleev/matrix-docker-ansible-deploy/pull/3696

Though I tried to make changes sensible as much as I could, I'm not quite convinced if they indeed make sense. Reviews would be appreciated pretty much. Also please note that the change is not tested!

### CODE REVIEW

File: roles/custom/matrix_playbook_migration/tasks/validate_config.yml [hunk 1]
```diff
@@ -223,6 +223,20 @@
           We found usage of the following variables: {{ matrix_playbook_migration_jitsi_migration_vars.keys() | join(', ') }}
       when: "matrix_playbook_migration_jitsi_migration_vars | length > 0"
 
+- when: matrix_playbook_migration_matrix_jitsi_migration_validation_enabled | bool
```
Comment (spantaleev) [inline roles/custom/matrix_playbook_migration/tasks/validate_config.yml:226]: This `when` variable should be adjusted

Comment (luixxiul) [reply roles/custom/matrix_playbook_migration/tasks/validate_config.yml]: Done → https://github.com/spantaleev/matrix-docker-ansible-deploy/pull/3698/commits/67205297ba40af3d8be471f157872a52e47b8818

**Comment by luixxiul** [comment]
I really hope this will not break anything...

**Comment by spantaleev** [comment]
With a bunch of additional last-minute fixes (see the commits above), I think it should be fine.

**Comment by luixxiul** [comment]
Thanks for taking care of it!

**Pull request merged**